src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -373,6 +377,32 @@ def _get_filter_conditions(conditions: Sequence[Condition]) -> Any:
     ).where
 
 
+def _parse_orderby(query: QueryDefinition) -> Optional[OrderBy]:
+    orderby = query.raw_orderby
+    if orderby is None:
+        return None
+
+    if "session.status" in query.raw_groupby:
+        raise InvalidParams("Cannot use 'orderBy' when grouping by sessions.status")
+
+    direction = Direction.ASC
+    if orderby[0] == "-":
```

Comment:
As you mention in the description, only one field is supported. Should we explicitly check that, and raise an explicit exception in that case?

Reply:
Sounds good.